build.gradle

```diff
@@ -84,6 +84,7 @@ dependencies {
       testCompile sourceSets.main.output
       testCompile sourceSets.api.output
       testCompile sourceSets.cli.output
+      testCompile sourceSets.translate.output
```

Comment:
Also not sure what this is for?

Reply:
@wpride Approving the changes but if this isn't needed then just remove it before you merge

Reply:
This is necessary because the `translate` files (specifically `XFormPlayer`) are now under test